docs/product/data-management-settings/filtering/index.mdx

```diff
@@ -35,7 +35,7 @@ If you had a legacy browser filter on before the update, the old filter will app
 
 ### Browser Extension Errors
 
-Some browser extensions are known to cause errors. You can use the browser extension filter to filter these out. The filter checks the error message and the event source to see if it's a known error coming from a browser extension.
+Some browser extensions are known to cause errors. You can use the browser extension filter to filter these out. The filter checks the error message and the event source to see if it's a known error coming from a browser extension. To see the full list of errors filtered by the browser extension filter, see the [source code in relay](https://github.com/getsentry/relay/blob/master/relay-filter/src/browser_extensions.rs#L9-L76).
```

Comment:
if we add more in the future , is the expectation that we come and update this link since rn it only goes up until line 76? if not, maybe just link to line 9?